### ISSUE TITLE
return of UNet3+ with deep supervision and CGM has no classification output for BCE

### ISSUE THREAD

**Issue by mvedang** [ZJUGiveLab/UNet-Version#3]
In the paper, it is mentioned that binary crossentropy is used to train the classifier, but in the code, there is no such provision. return statement returns only multiplication of deep supervision output. Also, why is a two-channel tensor necessary when instead it could also be a single channel tensor? I do not understand why argmax is used instead. What will the BCE loss expect in ground truth as there are two values when the the classification is just binary 0/1?

Really exciting paper and implementation otherwise. Really looking forward to implementing it in my applications.

With Best Regards,
Vedang

**Comment by babbu3682** [comment]
Same question, I am really curious about that. Why do they use 2 tensors in CGM module.

**Comment by xiaoerlaigeid** [comment]
Same Question

**Comment by wqchb1234** [comment]
Same Question。

**Comment by qinhaiyu218** [comment]
Same Question
